pkgs/development/python-modules/sphinx-jsonschema/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  docutils,
+  requests,
+  jsonpointer,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-jsonschema";
+  version = "1.19.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sphinx-jsonschema";
+    sha256 = "sha256-sjhf4ces8udZFSrv7QyxfJIGRbKnXJk0AAycUo59U8E=";
+  };
+
+  propagatedBuildInputs = [
+    docutils
+    requests
+    jsonpointer
+    pyyaml
+  ];
+
+  meta = with lib; {
+    description = "A Sphinx extension to display a JSON Schema";
+    homepage = "https://github.com/lnoor/sphinx-jsonschema";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [
+      JulianFP
+    ];
+  };
```

Comment:
```suggestion
  meta = {
    description = "A Sphinx extension to display a JSON Schema";
    homepage = "https://github.com/lnoor/sphinx-jsonschema";
    license = lib.licenses.gpl3Only;
    maintainers = with lib.maintainers; [
      JulianFP
    ];
  };
```

Reply:
I know that with lib is discouraged on the top of a derivation or module, but is that that true for small attribute sets as well? Just asking because I have seen that being done like this a lot and I want to make sure that I understand this right for the next time :D

Reply:
See #208242. I think it might also impact evaluation performance slightly.

Reply:
I see, thanks for the link. Removed the with lib now